src/jit/codegencommon.cpp

```diff
@@ -10265,7 +10265,52 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 /*****************************************************************************
  *
- *  Generates code for an EH funclet prolog.
+ *  Generates code for an EH funclet prolog. WIP.
+ *
+ *  Funclets have the following incoming arguments:
+ *
+ *      catch/filter-handler: ebp+8 = CallerSP, ebp+12 = the exception object that was caught (see GT_CATCH_ARG)
```

Comment:
It seems that the following code in ``exceptionhandling.cpp`` invokes funclets:
```
   //
    // Invoke the funclet.
    //
    dwResumePC = pfnHandler(sf.SP, OBJECTREFToObject(throwable));
```

``pfnHandler`` is of type ``HandlerFn *``:
```
src/ $ grep -Rn 'pfnHandler'
vm/exceptionhandling.cpp:3241:    HandlerFn*          pfnHandler = (HandlerFn*)uHandlerStartPC;
```

```
src/ $ grep -Rn 'HandlerFn'
...
vm/exceptionhandling.h:35:typedef DWORD_PTR   (HandlerFn)(UINT_PTR uStackFrame, Object* pExceptionObj);
```

There is no calling convention annotation on ``HandlerFn`` which means that ``CDECL``.